docs/search-views/semantic-search/index.md

```diff
@@ -1,4 +1,31 @@
 ---
 sidebar_position: 1
-title: Semantic Search Views
+title: Semantic Search Views - Beta
 ---
+
+Semantic search is a technique that allows searches based on the intent and contextual meaning of search phrases, rather than just keyword matching. It understands the relationships between words and concepts to provide more relevant results.
```

Comment:
I think I gave this comment earlier (probably on a different PR). What you describe here is indeed correct description of semantic search, but I am concerned that we are not doing most of what is described here in 17.17. As a user I might feel let down by the actual score of 17.17 after reading this. Should we tone it down and focus more on similarity search which we are doing in 17.17?

Reply:
Yes, we absolutely should. We should only talk about what we can do right now. We can expand the description later, as we add functionality.

Reply:
ok, can you then modify this description to focus more on the similarity search?

Reply:
Literally nowhere in the UI do we talk about similarity search, so it's a little awkward to introduce a brand new concept here. Are we planning to add other searches besides similarity in the next release or two?

Reply:
You may not need to introduce a new term here, you can just say that in 17.17 the semantic search feature aims to find similarities between entities (or something like that) -  We can't avoid introducing the concept of similarity because that is essentially what we are doing here.

Reply:
I like that! Added a note, please let me know if that works for you.